test/testcafe/spec/SelectAuthenticatorForEnroll_spec.js

```diff
@@ -76,6 +76,13 @@ test.requestHooks(mockEnrollAuthenticatorPassword)('should load select authentic
   await t.expect(selectFactorPage.getFactorDescriptionByIndex(4))
     .eql('Okta Verify is an authenticator app, installed on your phone or computer, used to prove your identity');
 
+  await t.expect(selectFactorPage.getFactorLabelByIndex(6)).eql('Atko Custom On-prem');
```

Comment:
Not for this PR, just a note for myself: we need to add a similar test for google authenticator